.github/pull_request_template.md

```diff
@@ -5,27 +5,22 @@
   -- Remember: assign a reviewer, or use @mentions if org. member.
   -->
 
-
-## Other information
-
-<!-- Other relevant info, e.g., before/after screenshots -->
-
-
 ## Checklist
 
 Tick *relevant* boxes, this PR is-a or has-a:
 
 - [ ] Bugfix
-  - [ ] Regression tests
```

Comment:
I agree with all of the other changes, except this one.  We should keep reminding ourselves to consider adding regression tests for weird bugs we don't want to see  again.